### DISCUSSION TITLE
Passenger transport: Evenly distributed vehicles

### DISCUSSION THREAD

**Discussion by Frozen-Rainbow** [voxeltycoon/issues#381] [General]
When adding multiple vehicles to a passenger route the vehicles all drive directly behind each other. It would be better if they manage themself to get the best distance between each other.

**Comment by JezzyBelle** [comment]
I can understand your feeling on this but actually we have tools to help us on this. They may not be complete or totally what you are looking for but you do have way more control that you think: 

You can set of create orders to: 
A: Wait til full   <---- can help on lower volume stations OR really slow things down. Hope they implement a percentage full.
B: Set order to Wait at station a certain amount of time. It doesn't mean they can't catch up again, but they end up with a more staggered and regulated flow FROM the station.

One thing still missing is a "wait til half full" or similar capacity trigger for buses. That would also help with some staggering and reduce the extended "wait til full" requests. Train cargo has this option due to multiple containers. Buses only effectively have one container so is not a current option. Trains tend to have to Fill to capacity in real life but buses have to keep moving to keep the connections viable and passengers happy. Cities/Passengers do not care about your profitability or ridership, just the overall service provided. YOu do not see buses typically waiting til they are full. They just go according to a scheduled or overall level of service provided

Actually, I am going to post a separate idea related that may change all of this IF implemented. Will call it "Passengers as a Service".


Bus service is a service, is usually granted by authority to a transport provider, and is fired IF they cannot meet certain service gurentees regardless if all or any lines themselves are profitable.
